pkgs/by-name/gi/git-autoshare/package.nix

```
@@ -0,0 +1,30 @@
+{ fetchFromGitHub
+, lib
+, python3Packages
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "git-autoshare";
+  version = "1.0.0b6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "acsone";
+    repo = "git-autoshare";
+    rev = version;
+    hash = "sha256-F8wcAayIR6MH8e0cQSwFJn/AVSLG3tVil80APjcFG/0=";
+  };
+
+  build-system = with python3Packages; [ setuptools-scm ];
+  dependencies = with python3Packages; [ appdirs click pyyaml ];
+
+  # Tests require network
+  doCheck = false;
+
+  meta = with lib; {
+    changelog = "https://github.com/acsone/git-autoshare/releases/tag/${version}";
+    description = "A git clone wrapper that automatically uses --reference to save disk space and download time.";
```

```suggestion
    description = "Git clone wrapper that automatically uses --reference to save disk space and download time.";
```
from the [meta attributes](https://github.com/NixOS/nixpkgs/tree/dc13fa9f319bae97dcfdf6a894bb324c0cfb594d/pkgs#meta-attributes) section of the contributing guide

> meta.description must:
>   - Be short, just one sentence.
>   - Be capitalized.
>   - ***Not start with the definite or an indefinite article.***